airflow/www/api/experimental/endpoints.py

```diff
@@ -58,8 +58,9 @@ def decorated(*args, **kwargs):
 
 def add_deprecation_headers(response: Response):
     """
-    Add `Deprecation HTTP Header Field
-    <https://tools.ietf.org/id/draft-dalal-deprecation-header-03.html>`__.
+    Add `Deprecation HTTP Header Field.
+
+    See also: <https://tools.ietf.org/id/draft-dalal-deprecation-header-03.html>`__.
```

Comment:
There’s a link syntax that’s broken here

Reply:
Well spotted, thanks.

Turns out the link is dead.  I did a search to find the new home of what should be there and it looks like it's an abandoned proposal: https://datatracker.ietf.org/doc/draft-dalal-deprecation-header/

I can update the link, but maybe we should remove the link entirely?